plugins/kubernetes/app/views/kubernetes/namespaces/index.html.erb

```diff
@@ -12,7 +12,10 @@
 
       <% @kubernetes_namespaces.each do |namespace| %>
         <tr>
-          <td><%= namespace.name %></td>
+          <td>
+            <%= namespace.name %>
+            <%= additional_info simple_format(namespace.comment) if namespace.comment? %>
```

Comment:
why isn't comment just a column in the table?

Reply:
because it could be pretty big and also noisy to ahve it in the index view